python/cuml/_thirdparty/sklearn/preprocessing/_discretization.py

```diff
@@ -357,9 +356,8 @@ def inverse_transform(self, Xt):
             idxs = np.asnumpy(Xinv[:, jj])
             Xinv[:, jj] = bin_centers[idxs.astype(np.int32)]
 
-        if not sparse_input:
-            # Dense input -> Dense output in correct format
-            return to_output_type(Xinv, output_type)
-        else:
-            # Sparse input -> Dense CuPy output
-            return Xinv
+        return Xinv
+
+        def _more_tags(self):
+            return {'X_types_gpu': ['2darray', 'sparse'],
+                    'X_types': ['2darray', 'sparse']}
```

Comment:
Same comment, just use the sparse mixin https://github.com/rapidsai/cuml/blob/branch-0.19/python/cuml/common/mixins.py#L306